types/d3-geo/index.d.ts

```diff
@@ -804,7 +804,62 @@ export interface GeoProjection extends GeoStreamWrapper {
      * @param object A geographic geometry collection supported by d3-geo (An extension of GeoJSON geometry collection).
      */
     fitSize(size: [number, number], object: ExtendedGeometryCollection<GeoGeometryObjects>): this;
-
+    /**
```

Comment:
Can you add a blank line between the `fitSize`, `fitWidth`, and `fitHeight` related definitions? Not critical, but the sectioned off readability going forward is a bit better. Seeing that d3-geo has very lengthy definitions.